### ISSUE TITLE
Send status check messages after merging

### ISSUE THREAD

**Issue by xt0rted** [integrations/slack#398]
If I have a CI build that runs on PRs as well as on master only the build status of PRs is sent into slack. Once the PR is merged you'll receive a message about the merge, another about the commits just pushed to master, but there's no status message about the CI build that just kicked off. It's possible that the PR build could pass but then fail once merged, or for someone to push right to master, and then you won't see if something fails.

**Comment by bkeepers** [comment]
Thanks for the feature request @xt0rted. I can see how that would be nice to have.
